pkg/dom0-ztools/rootfs/etc/containerd/config.toml

```diff
@@ -1,33 +1,23 @@
-version = 2
-state = "/run/containerd"
+version = 3
 root = "/persist/containerd-system-root"
 disabled_plugins = [
     "io.containerd.snapshotter.v1.btrfs",
     "io.containerd.snapshotter.v1.aufs",
     "io.containerd.internal.v1.opt",
     "io.containerd.internal.v1.tracing",
-    "io.containerd.tracing.processor.v1.otlp"
+    "io.containerd.tracing.processor.v1.otlp",
+    "io.containerd.runtime.v1.linux",
 ]
 
-[plugins]
-  [plugins."io.containerd.grpc.v1.cri"]
```

Comment:
Don't you need to declare the `runtime_type = "io.containerd.runc.v2"` somewhere? And the address needs to be set? Or are you relying on the defaults?

Reply:
> configs are only diff from default config

True. I tend to like being explicit with these things. But if you say it works, I won't complain.